python/tk_multi_publish2/api/manager.py

```diff
@@ -0,0 +1,446 @@
+# Copyright (c) 2018 Shotgun Software Inc.
```

Comment:
This object inherits much of what https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/python/tk_multi_publish2/processing/plugin_manager.py did. The implementation is different though.